### DISCUSSION TITLE
Reinstalling the OS without restoring Nextcloud data

### DISCUSSION THREAD

**Discussion by hunhejj** [nextcloud/all-in-one#3761] [Questions] [answered]
The data of Nextcloud is stored on an external drive configured through `NEXTCLOUD_DATADIR`. 
Now I would like to reinstall the OS, but obviously the data on the external drive will remain unaltered.

Is there an easy way to “move” my current running Nextcloud-AIO instance to continue using the existing data at the exact same location as before _without_ the need of restoring a full backup?

**Comment by szaimen** [comment, marked as answer]
Hi, I guess you are looking for this: https://github.com/nextcloud/all-in-one#how-to-store-the-filesinstallation-on-a-separate-drive

**Reply by hunhejj** [reply]
If I only wanted to move the nextcloud installation to the new system, but not other docker instances, would I then move `var/lib/docker/volumes/nextcloud_aio_*` only @szaimen ? How would I then install it on the new system? Would I run the `nextcloud/all-in-one` docker instance, run the initial configuration with dummy values so all instances are created, then stop the instances and replace with the "old" volumes? Or could that break anything?

**Reply by szaimen** [reply]
I would recommend to either use the above instructions or restore the instance using the backup archive.

**Reply by hunhejj** [reply]
If I were to restore the backup archive to a location where the files already exist, would it still speed up things or will it first erase everything from the target location anyway @szaimen ?

**Reply by szaimen** [reply]
It will use rsync to restore files incremental from the backup archive. So in fheory it will be faster...

**Reply by hunhejj** [reply]
Thank you for your advises @szaimen, I restored the backup and indeed it went pretty fast, for sure it did "reuse" the existing files.
